src/horizon/server.ts

```diff
@@ -874,7 +875,17 @@ export class Server {
   }
 }
 
-export namespace Server {
+/**
+ * Options for configuring connections to Horizon servers.
+ * @typedef {object} Options
+ * @memberof module:Horizon.Server
+ * @property {boolean} [allowHttp] Allow connecting to http servers, default: `false`. This must be set to false in production deployments! You can also use {@link Config} class to set this globally.
+ * @property {string} [appName] Allow set custom header `X-App-Name`, default: `undefined`.
+ * @property {string} [appVersion] Allow set custom header `X-App-Version`, default: `undefined`.
+ * @property {string} [authToken] Allow set custom header `X-Auth-Token`, default: `undefined`.
+ */
+
+export namespace HorizonServer {
```

Comment:
This has no tangible side-effects to downstream users because of the aliasing in `index.ts`, right?

Reply:
Yes, in my local testing, it behaves like normal.

- I tested the `/lib` output by using `node` in my terminal window, and interacting there.
- I tested the `/dist` output by creating a dummy html page that added the `/dist/stellar-sdk.js` file in a `<script>` tag and interacting with `StellarSdk` in the browser console.

Those may not be the best ways to test, though, so if there's another method, I'm certainly down for that.

Reply:
Nope, that's exactly what I'd have done!